modules/core/src/iterators/make-stream/make-dom-stream.ts

```diff
@@ -34,6 +34,11 @@ export function makeStream<ArrayBuffer>(
             controller.close();
           } else {
             // TODO - ignores controller.desiredSize
+            // @ts-expect-error
+            // No overload matches this call.
+            // The last overload gave the following error.
+            // Argument of type 'ArrayBuffer' is not assignable to parameter of type 'ArrayBufferLike'.
+            // Type 'ArrayBuffer' is not assignable to type 'SharedArrayBuffer'.ts(2769)
```

Comment:
@ibgreen I have no idea why this is necessary. I don't know how it's related to changes in this PR

Reply:
It fails on CI too, so it's not just a difference on my machine: https://github.com/visgl/loaders.gl/runs/6217125587?check_suite_focus=true#step:5:202